certified-connectors/DocuSignDemo/apiDefinition.swagger.json

```diff
@@ -493,6 +493,74 @@
         "x-ms-visibility": "advanced"
       }
     },
+    "/accounts/{accountId}/envelopes/{envelopeId}/documents/{documentId}/documentsDownload": {
+      "get": {
+        "summary": "Get documents from envelope",
```

Comment:
check with Doug on the action naming convention. He might want you to say `an envelope`